toolkit/sidechain/pallet/src/lib.rs

```diff
@@ -53,9 +185,12 @@ pub mod pallet {
 	#[pallet::genesis_config]
 	#[derive(frame_support::DefaultNoBound)]
 	pub struct GenesisConfig<T: Config> {
+		/// Genesis UTXO of the Partner Chain. This value is immutable after genesis.
```

Comment:
It is just immutable. Everything is present in genesis and nothing happens before genesis.